services/py-api/src/database/repository/teams_repository.py

```diff
@@ -96,3 +96,15 @@ async def get_verified_registered_teams_count(self) -> int:
         # Ignoring mypy type due to mypy err: 'Returning Any from function declared to return "int"  [no-any-return]'
         # which is not true
         return await self._collection.count_documents({"is_verified": True})  # type: ignore
+    
+    async def fetch_team_by_name(self, team_name: str) -> Result:
```

Comment:
change the name to fetch_by_team_name to match the rest of fetch functions

Reply:
Fixed in commit a9a6b60a1c533cba63bd89fc5bee4362218c00f7